test-configs.yaml

```diff
@@ -1729,6 +1732,7 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - ltp-ipc
```

Comment:
Note: `jetson-tk1` is actually a Chromebook, but yes it's using U-Boot unlike most other more recent ones that are using Coreboot/Depthcharge.  It might be put in #520, but that's a detail.

Reply:
Tested OK with staging-mainline: https://staging.kernelci.org/test/plan/id/5fa2be45aff8d721fa59474f/